docs/source/development.rst

```diff
@@ -28,39 +28,35 @@ equivalent packages for your distribution:
 -  vagrant
 
 The only software actually required to run the playbooks is Ansible and its
-dependencies. To install the versions that have been tested as working with
-this system, use the requirements.txt file in the top of the project to install
-all the necessary software into a Python virtualenv. Alternatively, invoke the
-script bin/ensure\_virtualenv.sh to install the Python dependencies into the
-folder .venv/
+dependencies. The other packages listed above are required only to install and
+build Ansible and its dependencies, such as PyYAML. Thus, if you are looking
+to package Cinch for a new distribution, the above packages, less vagrant,
+are a good starting place for build dependencies.
 
 If installing manually, you can activate your Python virtualenv of choice and
-issue the command ``pip install -r requirements.txt``
+issue the command ``pip install .``. As a developer, if you plan to make
```

Comment:
Might want to include a path here to ensure that folks are pointed to the right place, like **pip install /path/to/cinch/rootdir** or tell the user to **cd** into the directory first.